integration-tests/utils/integration-utils.js

```diff
@@ -277,8 +277,17 @@ const logPageConsoleMessages = page => {
 }
 
 const logPageAjaxResponses = page => {
+  const terraRequests = [
+    'broad',
+    'terra',
+    'googleapis'
+  ]
```

Comment:
Filter out noises, log Terra application Fetch/XHR requests. Example of requests that won't be logged are `GET http://localhost:3000/static/media/covid-hero.1eabde8bceeaf1679703.jpg` or `http://localhost:3000/static/css/main.34135f2a.css`